packages/client/eslint.config.js

```diff
@@ -44,10 +43,10 @@ export default [
       // https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks
       // compat with ESLint v9, see https://github.com/facebook/react/issues/28313
       // recent "compatibility" PR does not address flat config at all, so still on "hacked" eslint config
+      // also need to use "rc" version - been rc for 5mths! - otherwise peer deps complaints
```

Comment:
I forgot about this when I migrated Alpaca - at some point, perhaps, they will release 5.1.0, but as the comment says it has been RC for a while now ..